docs/docs/code_of_conduct.md

```diff
@@ -10,4 +10,4 @@ Project maintainers have the right and responsibility to remove, edit, or reject
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by opening an issue or contacting one or more of the project maintainers.
 
-This Code of Conduct is adapted from the [Contributor Covenant](http:contributor-covenant.org), version 1.0.0, available at [http://contributor-covenant.org/version/1/0/0/](http://contributor-covenant.org/version/1/0/0/)
+This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org/), version 1.0.0, available at [https://www.contributor-covenant.org/version/1/0/0/code-of-conduct/](https://www.contributor-covenant.org/version/1/0/0/code-of-conduct/)
```

Comment:
I think we should remove this page entirely, it is not up-to-date and duplicate of https://github.com/vatesfr/xen-orchestra/blob/master/CODE_OF_CONDUCT.md

Reply:
Fixed in https://github.com/vatesfr/xen-orchestra/pull/8078/commits/85d68c926e19f7aee9667422277f83dabd764546